app/src/main/java/com/infomaniak/swisstransfer/ui/screen/newtransfer/importfiles/components/TransferTypeButtons.kt

```diff
@@ -40,22 +36,31 @@ import com.infomaniak.swisstransfer.ui.theme.Margin
 import com.infomaniak.swisstransfer.ui.theme.SwissTransferTheme
 import com.infomaniak.swisstransfer.ui.utils.GetSetCallbacks
 import com.infomaniak.swisstransfer.ui.utils.PreviewLightAndDark
+import kotlinx.coroutines.launch
 
 @Composable
 fun TransferTypeButtons(horizontalPadding: Dp, transferType: GetSetCallbacks<TransferTypeUi>) {
-    Row(
-        modifier = Modifier
-            .horizontalScroll(rememberScrollState())
-            .padding(horizontal = horizontalPadding),
-        horizontalArrangement = Arrangement.spacedBy(Margin.Mini),
-    ) {
-        for (transferTypeEntry in TransferTypeUi.entries) {
-            TransferTypeButton(
-                transferType = transferTypeEntry,
-                isActive = { transferTypeEntry == transferType.get() },
-                onClick = { transferType.set(transferTypeEntry) },
-            )
+
```

Comment:
Your preview is in a `Column` but this component is used in a `Row`. Is that correct ?

Reply:
It doesn't represent how we actually display the component but if we're ok with that, I'm good.